zenodo/modules/pages/views.py

```diff
@@ -51,39 +50,62 @@
 )
 def support():
     """Render contact form."""
-    form_class = contact_form_factory()
-    form = form_class()
+    form = ContactForm()
+    os, browser, device = user_agent_information()
```

Comment:
There's no need to unpack this tuple, since you always pass these parameters around as is. Instead of that you can  keep them in a `dict` and pass it.